samza-core/src/main/java/org/apache/samza/runtime/AbstractApplicationRunner.java

```diff
@@ -82,8 +82,14 @@ ExecutionPlan getExecutionPlan(StreamApplication app, String runId, StreamManage
     cfg.put(ApplicationConfig.APP_MODE, mode.name());
 
     // create the physical execution plan
-    ExecutionPlanner planner = new ExecutionPlanner(new MapConfig(cfg), streamManager);
-    return planner.plan(specGraph);
+    Config config = new MapConfig(cfg);
```

Comment:
trivial: there's already a member-variable named "config" in this class and it might be a bit confusing to disambiguate it with the local variable. maybe name it to something specific to clarify the usage here.